geth/jail/execution_policy.go

```diff
@@ -30,22 +113,62 @@ func NewExecutionPolicy(nodeManager common.NodeManager, accountManager common.Ac
 	}
 }
 
-// ExecuteSendTransaction defines a function to execute RPC requests for eth_sendTransaction method only.
-func (ep *ExecutionPolicy) ExecuteSendTransaction(req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
+// Execute handles the execution of a RPC request
+func (ep *ExecutionPolicy) Execute(req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
 	config, err := ep.nodeManager.NodeConfig()
 	if err != nil {
 		return nil, err
 	}
 
-	if config.UpstreamConfig.Enabled {
-		return ep.executeRemoteSendTransaction(req, call)
+	if params.SendTransactionMethodName == req.Method {
+		if config.UpstreamConfig.Enabled {
+			return ep.ExecuteRemoteSendTransaction(req, call)
+		}
+
+		return ep.ExecuteLocalSendTransaction(req, call)
+	}
+
+	var res *otto.Object
+	var resErr error
+	switch rpcCommandsRoute[req.Method] {
```

Comment:
1. No `default` case.
2. Each line could `return ...` immediately instead of having `var`s as nothing is done with these variables anyway.